src/main/java/seedu/address/model/score/Title.java

```diff
@@ -0,0 +1,50 @@
+package seedu.address.model.score;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents the score title in a score.
+ * Guarantees: immutable; is valid as declared in {@link #isValidTitle(String)}
+ */
+public class Title {
+
```

Comment:
Maybe instead of the title, the description is more proper.

Reply:
It depends on what we will put in our title/description. Are we going to put terms like "Midterm Assessment 1"?
Or are we going to have an actual description for it? 

Label might sound better?

Reply:
Yeah, the label sounds better. Just feel a score has a title a bit odd.